internal/graderd/docker_helper.go

```diff
@@ -1,22 +1,24 @@
 package graderd
```

Comment:
i think we should move this to `grader-task` instead since this is so specific to it.